extensions/stackdriver/common/utils.cc

```diff
@@ -22,6 +22,19 @@ namespace Extensions {
 namespace Stackdriver {
 namespace Common {
 
+namespace {
+
+const std::string getContainerName(
```

Comment:
nit: make return value `const std::string&` and make type of returned value (line 116) to be `auto&` to save a copy.

Reply:
we don't even need a function for this since it's used in one place only.